policyengine_us/variables/gov/territories/pr/tax/income/credits/low_income/pr_low_income_credit.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class pr_low_income_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Puerto Rico low income credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://casetext.com/statute/laws-of-puerto-rico/title-thirteen-taxation-and-finance/subtitle-17-internal-revenue-code-of-2011/part-ii-income-taxes/chapter-1007-credits-against-tax/subchapter-b-refundable-credits/30212-credit-for-low-income-individuals-older-than-sixty-five-65-years-of-age"
+    defined_for = "pr_low_income_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.territories.pr.tax.income.credits.low_income.amount
+        person = tax_unit.members
+        eligible_person = person(
+            "pr_low_income_credit_eligible_person", period
+        )
+        eligible_people = tax_unit.sum(eligible_person)
```

Comment:
Irrelevant with `pr_low_income_credit_eligible_people` but for future reference this can combine the two statements:
```python
        eligible_people = add(tax_unit, period, ["pr_low_income_credit_eligible_person"])
```